1.0/docs/devguide/styling.md

```diff
@@ -24,92 +40,148 @@ provided via `<style>` tags placed inside the element's local DOM `<template>`.
-          }
+
         </style>
 
-        <div id="child-element">In local DOM!</div>
-        <div class="content-wrapper"><content></content></div>
+        <div class="child-element">In local DOM!</div>
+        <content></content>
 
       </template>
 
       <script>
-
           Polymer({
               is: 'my-element'
           });
-
       </script>
 
     </dom-module>
 
-To place styles outside of the element, or share styles between elements, you can create
-a [style module](#style-modules).
+In the page, you might have:
+
+    <my-element>
+        <!-- In shady DOM the class is "child-element style-scope my-element" -->
+        <div class="child-element">In light DOM!</div>
+    </my-element>
+
+And see that only `In local DOM` has a yellow background.
 
 **Note:**  Prior to Polymer 1.1, the recommendation was to place `<style>` tags
 inside the `<dom-module>` for an element (but _outside_ the `<template>`). This
  is still supported, but is no longer recommended.
 {: .alert .alert-info }
 
+**NOTE: I like introducing what shadow DOM styling SHOULD be like, and then explain what we should do with shady DOM, and why.**
+
+#### Shim limitations
+
+There is no need to do any special workaround in this simple case, since shady
+DOM will automatically change the style as well as the element's class to make
+sure that styling in the local DOM doesn't leak out of `my-element` by adding a
+required class, `my-element`, to the created element (and to the selector).
+
+
+### Styling distributed children (`::content`)
+
+In shadow DOM, you can style an element's light DOM using the `::content` pseudo element. So, you can do this:
+
+    <dom-module id="my-element">
+
+      <template>
+
+        <style>
+          /* WARNING: This does NOT work in shady DOM */
+          ::content > .special {
+            background: orange;
+          }
+        </style>
 
-### Styling distributed children (::content)
+       <div class="special">This should not be affected</div>
+       <content></content>
+      </template>
 
-Under shady DOM, the `<content>` tag doesn't appear in the DOM tree. Styles are rewritten to remove the
-`::content` pseudo-element, **and any combinator immediately to the left of `::content`.**
+      <script>
+          Polymer({
+              is: 'my-element'
+          });
+      </script>
 
-This implies:
+    </dom-module>
 
-*   You must have a selector to the left of the `::content` pseudo-element.
 
-        :host ::content div
+In shadow DOM, the `.special` class will only ever affect elements in the light
+DOM. So when you use your element:
 
-    Becomes:
+    <my-element>
+        <div class="special">This should be special!</div>
+    </my-element>
 
-        x-foo div
+Only `This should be special!` will have an orange background.
 
-    (Where `x-foo` is the name of the custom element.)
+#### Shim limitations
 
-*   To limit styles to elements inside the ::content tag, add a wrapper element around the
```

Comment:
This deleted paragraph concisely describes why we're using the workaround starting at L133.